etc/codecoverage/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Application entry point
+ *
+ * Example - run a particular store or website:
+ * --------------------------------------------
+ * require __DIR__ . '/app/bootstrap.php';
+ * $params = $_SERVER;
+ * $params[\Magento\Store\Model\StoreManager::PARAM_RUN_CODE] = 'website2';
+ * $params[\Magento\Store\Model\StoreManager::PARAM_RUN_TYPE] = 'website';
+ * $bootstrap = \Magento\Framework\App\Bootstrap::create(BP, $params);
+ * \/** @var \Magento\Framework\App\Http $app *\/
+ * $app = $bootstrap->createApplication(\Magento\Framework\App\Http::class);
+ * $bootstrap->run($app);
+ * --------------------------------------------
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```

Comment:
Can we move copyright to top of file?

Reply:
This is where it's at in the original index.php from magento2ce.
I don't want to change it and introduce an unnecessary diff.
It matches CE.

Reply:
Ok. I did not know that